### DISCUSSION TITLE
Production ready

### DISCUSSION THREAD

**Discussion by MazzGuille** [microsoft/fluentui-blazor#1522] [General]
Hi! First of all, thanks for your awesome work on this library! I like how professional it looks, I am very curios (and sorry if this is the wrong place to ask) if this library is 100% production ready, I am about to start a new project for a client and would like to give it a try, please if anyone can confirm or deny, let me know, I currently use MudBlazor, but I'd love to have an alternative and this library looks like it can be a better fit for the project my client has requested.

**Comment by vnbaaij** [comment]
Hi,

Thanks for the compliment! 

Whether or not this is production ready, I'll leave for others to fill in but we believe it is. The .NET Team here at Microsoft is building the [Aspire Dashboard](https://learn.microsoft.com/en-us/dotnet/aspire/fundamentals/dashboard) with it and the library is also part of official Microsoft [Visual Studio Teams Toolkit](https://learn.microsoft.com/en-us/microsoftteams/platform/toolkit/toolkit-v4/teams-toolkit-fundamentals-vs?pivots=visual-studio-v17-7).

Just so you are aware...The Microsoft Fluent UI Blazor library is an open source project and is **not** an official part of ASP.NET Core, which means it’s **not** officially supported and isn’t committed to ship updates as part of any official .NET updates. It is built and maintained by Microsoft employees (and other contributors) and offers support, like most other open source projects, on a best effort base through the GitHub repository **only**.

Are we without bugs? No, of course not. Do we make mistakes/bad decisions sometimes in bug fixing or adding new features? Yes, we certainly do! But everything we do is done with the best intentions and we do our best to support you and your projects.

Moving this to the discussion area (as it is not really an issue) so others can chime in.

**Reply by MazzGuille** [reply]
Hi, thanks for the soon reply, I was so busy I forgot to check on this until now, didn't expect to get an answer so soon, thanks for the dedication and effort, I really like what you do!

**Comment by NickHirras** [comment]
My team is using this for a large-volume production application at our company. We have been happy with it. Where we have run across an issue, it was either addressed quickly by this team, or we were able to contribute a fix ourselves and submit a PR to this project. The people working on this project seem readily available to help with issues or suggestions.

**Reply by MazzGuille** [reply]
Good to know! I will give it a try for my next project (which starts on Monday) I am very happy to see replies so fast!
